core/src/sigverify_stage.rs

```diff
@@ -97,11 +97,12 @@ impl SigVerifyStage {
             (len as f32 / verify_batch_time.as_s())
         );
 
-        datapoint_debug!(
+        datapoint_info!(
```

Comment:
I think we need to accumulate these metrics, otherwise this will submit it too many times. That's why we have it as _debug by default.

Reply:
yeah, noticed a lot more loggings at info level. Looks like @jbiseda already has an PR to accumulate stats #19674. Should be able to merge with it

Reply:
restored this change in fav of Jeff's PR